src/main/java/seedu/duke/Duke.java

```diff
@@ -3,19 +3,36 @@
 import java.util.Scanner;
 
 public class Duke {
-    /**
-     * Main entry-point for the java.duke.Duke application.
-     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
 
+        welcomeMessage();
+
+        String input;
         Scanner in = new Scanner(System.in);
-        System.out.println("Hello " + in.nextLine());
+        input = in.nextLine();
+
+        boolean isBye = false;
+        while (!isBye) {
+            switch (input) {
+            case "bye":
+                isBye = true;
+                byeMessage();
+            default:
+                System.out.println(input + "\n");
+                input = in.nextLine();
+            }
+        }
+
+
+
+    }
+
+    private static void byeMessage() {
```

Comment:
Function name should be a verb, like printByeMessage

Reply:
Resolved!